.github/workflows/deployExpensifyHelp.yml

```diff
@@ -66,3 +66,10 @@ jobs:
         with:
           token: ${{ secrets.OS_BOTIFY_TOKEN }}
           body: ${{ format('A preview of your ExpensifyHelp changes have been deployed to {0} ⚡️', steps.deploy.outputs.alias) }}
+      
+      - name: Reindex google search
+        if: ${{ github.event_name == 'push' }}
+        uses: fjogeleit/http-request-action@v1
```

Comment:
Can we just use `curl` or `wget` or something instead of this community action?

Reply:
yes def! i forgot that the action runs on linux 🙂